utils/clflags.ml

```diff
@@ -420,26 +420,42 @@ module Compiler_pass = struct
      - the manpages in man/ocaml{c,opt}.m
      - the manual manual/manual/cmds/unified-options.etex
   *)
-  type t = Parsing | Typing
+  type t = Parsing | Typing | Scheduling
 
   let to_string = function
     | Parsing -> "parsing"
     | Typing -> "typing"
+    | Scheduling -> "scheduling"
 
   let of_string = function
     | "parsing" -> Some Parsing
     | "typing" -> Some Typing
+    | "scheduling" -> Some Scheduling
     | _ -> None
 
   let rank = function
     | Parsing -> 0
     | Typing -> 1
+    | Scheduling -> 50
 
   let passes = [
     Parsing;
     Typing;
+    Scheduling;
   ]
-  let pass_names = List.map to_string passes
+  let is_compilation_pass _ = true
```

Comment:
I think I would prefer to introduce the function after it becomes actualy useful.

Reply:
Actually, this tautology was already there. I just moved all of the checks to one place. 

I kept it to make sure that future passes explicitly to consider the interaction with Clflags.compilation_only, which wasn't obvious to me.  If you feel that it's not worth the extra clutter, I'm happy to remove it.  The only passes I can think of that could return false are link-time optimizations we don't have yet.

Reply:
Ok, let's keep it if it was already here.